tool/grind.dart

```diff
@@ -5,8 +5,28 @@ import 'dart:io';
 
 import 'package:grinder/grinder.dart';
 
-main(args) => grind(args);
+import 'package:unscripted/unscripted.dart';
 
+main(args) {
+
+  addTask(new GrinderTask(
+      'bump',
+      taskFunction: () {
+        var releaseType = context.invocation.positionals.first;
```

Comment:
Can we make accessing the args more terse, or add aliases to help references to the args be terse?

```
context.args.rest.first;
context.args.first;
context.args['foo];

args.rest.first;
args['foo'];
```

thoughts? This: `context.invocation.positionals.first` strikes me as pretty long, given that it may be the most common use of task args.